tests/select_expr.rs

```diff
@@ -521,8 +521,8 @@ fn test_select() {
         2
     }
 
-    let table = sql!(TableSelectExpr[result()]).unwrap();
-    assert_eq!(id3, table.id);
+    //let table = sql!(TableSelectExpr[result()]).unwrap();
```

Comment:
This issue is caused by an hygiene issue.
To fix it, I would use `Span::def_site()` [here](https://github.com/antoyo/tql/pull/37/files#diff-a53c40b58bc3ef441906076c9cf10acaR59) but it's not stable yet.
So instead, could you please prefix every generated identifiers by `__tql_` to avoid any name clash?
Thanks.